package/src/skia/web/JsiSkCanvas.ts

```diff
@@ -78,12 +87,12 @@ export class JsiSkCanvas
     paint?: SkPaint | null
   ) {
     this.ref.drawImageCubic(
-      toValue(img),
+      JsiSkImage.fromValue(img),
       left,
       top,
       B,
       C,
-      toOptionalValue(paint)
+      paint ? JsiSkPaint.fromValue(paint) : paint
```

Comment:
Should these be written like this instead to make it a bit clearer what's going on?

```ts
paint ? JsiSkPaint.fromValue(paint) : null
```

Reply:
here it can be null or undefined, and I wanted to keep it exactly this way. CanvasKit may not make any difference between the two but I don't want to make any assumptions here

Reply:
Yes, it can be null/undefined - but the way this is written makes that a bit unclear. You've also used the pattern I suggested other places in this PR so I think you should use the same pattern here. See f.ex. `JsiSkFont` on line 51 - same "problem" but much more easy to read.

Reply:
No, I think writing it this way is a lot clearer :) That was my point with the initial comment. But as I said this is not a show stopped for approving this PR so I you don't agree I'll approve it anyways.